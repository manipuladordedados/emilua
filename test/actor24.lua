if _CONTEXT ~= 'main' then
    return
end

local ch = spawn_vm('.')

local m = {}
m.m = m
ch:send(m)
