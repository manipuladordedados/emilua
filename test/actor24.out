Main fiber from VM 0x1 panicked: 'Reference cycle exists'
stack traceback:
	[string "?"]: in function 'send'
	input.lua:9: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
